lab9/models/digital_element.py

```diff
@@ -0,0 +1,5 @@
+class DigitalElement:
```

Comment:
I believe this class is meant to be abstract, so it should inherit from [abc.ABC](https://docs.python.org/3/library/abc.html).